Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -137,12 +146,21 @@ protected static function getTempFile()
      *
      * @param string           $testFileName Filename
      * @param string           $testTitle    Test title
+     * @param string[]         $config       Test file configuration
```

Comment:
if I get it right, the config have only requirements ?


also, `$config['hhvm']` is compared to bool, so the typehinting here is wrong